meetings/2022-03-08.md

```diff
@@ -0,0 +1,114 @@
+
+* Pavlik: mostly applies to data registries, e.g. in pod browser case where something should have access to more than a specific shape tree
+* Justin: Few use case would be a good place to start. 
+* Pavlik: In the spirit of discussion with Jeff last week, look at how solidOS and other apps can more readily adopt interop patterns
+ 
+#### Proposal for publicly accessible data
+
+[Issue#243](https://github.com/solid/data-interoperability-panel/issues/243)
+
+* Justin: We need to build on what Laurens put in that issue.
+* Laurens: I can propose something there based on what's in the spec
+
+#### [Incorporate profile requirements from solid-oidc](https://github.com/solid/data-interoperability-panel/issues/210)
+
+* Justin: I actually think we have any blockers left here. In sai-java i have it implemented. This is a matter of reflecting it in the spec. We have now JSON-LD context for the spec itself and patter for how those contexts can be combined.
+* Pavlik: I think we should consider again whether we want to mandate that context because we don't guarantee on solid that the compacted version is preserved and it becomes more complicated to store it. I think it's problematic that it couldn't be reliably stored in solid storage.
+* Justin: That requirement is imposed in solid-oidc document. In sai-java the code takes content type for read and write. It's not really burden on the client. The rational of not having to deal with with RDF on the OP?
+* Pavlik: Yes i think the rationale is for OPs that aren't RDF aware it's easier for them just to parse JSON. Generally believe that it's better for consumer to provide context they rely on for compaction and framing.
+
+#### [Determine how to dynamically authorize access to reference lists on inherited instances](https://github.com/solid/data-interoperability-panel/issues/174)
+
+* Justin: There is an open question on weather we want to go into in the interop spec. Or is it more about tooling in companion specs like shape trees.
+* ...: The problem to solve is related to `Inherited` Data Grants. For example you have project you get access to `project-1` and based on the shape tree reference you get access to related issues and milestones. This is one of the real powers that we have based on shape trees and shape tree references. We model it with `Inherited` grant scope, access to issues is inherited based on access to the project.
+* ...: Not all tasks in data registration belong to `project-1`, semantically it is pretty straight forward. Given project will have a graph link to issues that are part of it. Our access control systems don't provide mechanism to give access based on the presence of some graph relationship. I think they should but they don't at this moment.
+* ...: We actually have use cases in authorization panel. In long term it would be good to get access control system supporting it. Meanwhile we can have notifications and for example AA setting access based on those relations.
```

Comment:
```suggestion
* ...: We actually have use cases in authorization panel. In long term, it would be good to get access control system supporting it. Meanwhile, we can have notifications and, for example, AA setting access based on those relations.
```